using-kite-properties.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Using Kite Properties
+---
+
+Kite gives you the option of changing settings of existing system properties to enhance performance. You also have the option of creating custom properties.
+
+## Setting Kite Properties
+
+There are several dataset properties that affect how files are written. These settings are especially relevant for Parquet because it buffers records in memory. 
+
+`kite.write.cache-size` controls the number of files kept open by an HDFS or Hive dataset writer.
```

Comment:
Could you restructure so both properties here are separate sections? I think that would be more clear.

Also, I'd make a sub-section of the parquet.block.size section called "Avoiding Parquet OutOfMemoryExceptions".